src/core/ui/forms/FormikFileInput/FormikFileInput.tsx

```
@@ -1,63 +1,68 @@
     }
   };
 
   return (
     <FormikInputField
       name={name}
       loading={!storageConfig}
-      onBlur={checkProtocol}
       endAdornment={
         storageConfig &&
-        storageConfig.canUpload && (
+        storageConfig.canUpload &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)